.github/workflows/codeRiewson.yml

```
@@ -14,33 +14,52 @@ jobs:
         with:
           fetch-depth: 0  # Ensure the whole repo history is pulled
 
+      - name: Install jq
+        run: sudo apt-get update && sudo apt-get install jq
+
       - name: Set up Node.js
         uses: actions/setup-node@v3
```

actionlint (1.7.4)

the runner of "actions/setup-node@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)